configurations.yml

```diff
@@ -0,0 +1,19 @@
+logging:
+  log_file: "./logs/out_{time}.log"
+  format: "{time:YYYY-MM-DD at HH:mm:ss} {level} {message}"
+  backtrace: true
+  diagnose: true
+  level: "DEBUG"
+  retention: "7 days"
+
+paths:
+  census_path: "./path/to/census_data"
+  gnaf_path: "./path/to/gnaf_data"
+  shapefile_path: "path/to/shapefile.shp"
+
+
+crs: "EPSG:7844"
```

Comment:
A little comment on what this means would help

Reply:
Added comments throughout configurations.yml